src/pages/agent/methods-table/coverage-cell/coverage-cell.tsx

```diff
@@ -32,7 +32,7 @@ const Content = styled.div(({ showCoverageIcon } : { showCoverageIcon: boolean }
 
 export const CoverageCell = ({ value = 0, showCoverageIcon }: { value: number, showCoverageIcon: boolean }) => (
   <Content showCoverageIcon={showCoverageIcon}>
-    <span data-test="coverage-cell:coverage">{`${percentFormatter(value)}%`}</span>
+    <span data-test="coverage-cell:coverage" title={`${percentFormatter(value)}`}>{`${percentFormatter(value)}%`}</span>
```

Comment:
```suggestion
    <span data-test="coverage-cell:coverage" title={coverage}>{`${coverage}%`}</span>
```
you can calc it once